test/cq-overflow.c

```diff
@@ -525,21 +525,17 @@ int main(int argc, char *argv[])
 	}
 
 	unlink(fname);
-	if(vecs != NULL)
```

Comment:
These style cleanups are good, but should not be bundled with any kind of code or functional change.